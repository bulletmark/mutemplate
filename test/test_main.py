#!/usr/bin/env python
from templates import Template
print(Template('toplevel').render(n=4), end='')
