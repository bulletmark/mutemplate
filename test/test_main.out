==== Table of squares with default n:
| 1 |   1 |
| 2 |   4 |
| 3 |   9 |
| 4 |  16 |
====

==== Table of squares with n=n*2:
| 1 |   1 |
| 2 |   4 |
| 3 |   9 |
| 4 |  16 |
| 5 |  25 |
| 6 |  36 |
| 7 |  49 |
| 8 |  64 |
====

==== Table of squares dynamic name and n=n*3:
| 1 |   1 |
| 2 |   4 |
| 3 |   9 |
| 4 |  16 |
| 5 |  25 |
| 6 |  36 |
| 7 |  49 |
| 8 |  64 |
| 9 |  81 |
| 10 | 100 |
| 11 | 121 |
| 12 | 144 |
====

==== For loop:
1 is odd
2 is even
3 is odd
4 is even

==== While loop:
5
4
3
2
1

==== Include child/child template:
| 1 |   1 |
| 2 |   4 |
| 3 |   9 |
| 4 |  16 |

==== Empty file:
====
